gcc/rust/ast/rust-ast.h

```diff
@@ -1333,6 +1333,8 @@ class InherentImplItem
 
   virtual void mark_for_strip () = 0;
   virtual bool is_marked_for_strip () const = 0;
+
+  virtual Location get_locus_slow () = 0;
```

Comment:
Nitpick - this should be const-qualified.

Reply:
Thanks for the review i will fix these before merge